src/features/unified-share-modal/utils/computeSuggestedCollabs.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @flow
+ * @file Suggested Collaborator utility functions
+ * @author Box
+ */
+import type { SuggestedCollab, SuggestedCollabLookup } from '../flowTypes';
+import fuzzySearch from '../../../utils/fuzzySearch';
+
+/**
+ * Function to compute suggested collaborators given a list of contacts and cached suggested collaborators.
+ * Used by input components to help augment API results with cached suggestions using a fuzzy search.
+ *
+ * @param {Array<C>} contacts Contacts returned from the server with exact matches
+ * @param {SuggestedCollabLookup} suggestedCollaborators Cached suggested collab lookup to fuzzy match on
```

Comment:
Get rid of the jsdocs, we dont add them anymore as people rarely update them or read them. Types and arg names should be self documenting.